build.sh

```diff
@@ -27,14 +27,18 @@ function save_build_info() {
     cd .. || exit 1
 
     cat <<-EOF > run/Defaults
-#   File that lists how the build (binaries) were made
+#   This file lists how the build (binaries) were made
 [Build Configuration]
 System Wide Build=Yes
 Architecture="$(uname -m)"
 OpenMP, OpenCL=No
```

Comment:
This is technically wrong (the OpenCL part), but I don't know if it will work for end users.